src/charm.py

```diff
@@ -84,7 +84,7 @@ def _zookeeper_layer(self) -> Layer:
                     "summary": "zookeeper",
                     "command": self.zookeeper_config.zookeeper_command,
                     "startup": "enabled",
-                    "environment": {"KAFKA_OPTS": " ".join(self.zookeeper_config.kafka_opts)},
+                    "environment": {"SERVER_JVMFLAGS": " ".join(self.zookeeper_config.kafka_opts)},
```

Comment:
Could we rename `zookeeper_config.kafka_opts` to `zookeeper_config.server_jvmflags`?